packages/viewer/src/components/viewer-pin-label/vertex-pin-label.css

```diff
@@ -0,0 +1,34 @@
+:host {
+  /**
+   * @prop --viewer-annotations-pin-label-background-color: A CSS color that
+   * specifies the color of the label background
+   */
+   --viewer-annotations-pin-label-background-color: var(--blue-200);
+
+  /**
+   * @prop --viewer-annotations-pin-label-background-color: A CSS color that
+   * specifies the color of the label background
+   */
+   --viewer-annotations-pin-label-border-color: var(--blue-600);
+
+  overflow: hidden;
+  pointer-events: none;
+}
+:host([mode='']) .pin-label {
+  color: var(--white);
+  background: var(--neutral-900);
+}
```

Comment:
Might be nice to support customizing these values in case I changed the `pin-label-background-color`/`pin-label-border-color` to white/neutral-900